rollupConfigFactory.js

```diff
@@ -0,0 +1,214 @@
+import fs from 'fs';
+import path from 'path';
+import replace from 'rollup-plugin-replace';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import istanbul from 'rollup-plugin-istanbul';
+import pkg from './package.json';
+import Visualizer from 'rollup-plugin-visualizer';
+
+/**
+ * browser-friendly UMD build
+ * @param {string} dirName Output destination directory
```

Comment:
nice details about jsdoc, I haven't been using it on this repo, but it's always welcome 👍